docs/usage/getting-started/running.md

```diff
@@ -1,70 +1,88 @@
-When you install Renovate from npm it naturally does not come bundled with any third-party tools or languages such as Ruby, Python, Composer, Bundler, Poetry, etc.
+You can run this package in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
+
+Renovate's npm package does not bundle any third-party tools or languages like Ruby, Python, Composer, Bundler, Poetry, etc.
 If you need Renovate to support any non-npm lock files like Bundler then you'll need to make sure all required third-party tools are pre-installed in the same environment alongside Renovate before you run it.
 
 The `renovate` npm package is compatible with all of Renovate's supported platforms.
 
-Renovate requires Node.js `>=14.15.0` and Git `>=2.33.0`.
+Renovate requires Node.js `>=18.12.0` and Git `>=2.33.0`.
+
+#### Docker images
+
+Renovate is also distributed as Docker images on Docker Hub (`renovate/renovate`) and GitHub container registry (`ghcr.io/renovatebot/renovate`).
+These Docker images work on all the hosting platforms that Renovate supports.
+
+You must run the Docker images in a `linux/amd64` container.
+We're working on `linux/arm64` container support.
+You can't run the Docker images on a Windows or macOS container.
+
+In general, you can run Renovate on Windows with global mode.
 
-#### Docker image
+There are two flavors:
 
-The `renovate` npm package is also distributed via pre-built Node.js images on Docker Hub (`renovate/renovate`).
+- The default "full" image, this is the image you'll get if you use the `latest` tag
+- The `-slim` image
 
-The `slim` image contains only Node.js so works if either:
+##### The default image
 
-- You do not require any additional package managers, or
-- You map the Docker socket into the container so that Renovate can dynamically invoke "sidecar" images when necessary
+The default image comes with every package manager that Renovate supports.
+This makes the default image the best choice for most users.
 
-The "full" image (which `latest` defaults to) has every package manager which Renovate supports already preinstalled.
-This approach works best for many, but does have the following downsides:
+The default image has some downsides, because it:
 
-- It only has _one_ version of each language/manager - usually the latest
-- It's several gigabytes in size
+- Comes pre-installed with _one_ version of each language/manager - usually the latest
+- Weighs several gigabytes
 
-The `renovate/renovate` Docker images are compatible with all of Renovate's supported platforms.
+##### The slim image
+
+The `-slim` image only comes with the Node.js environment, so it works when:
+
+- you don't need any additional package managers, or
+- you map the Docker socket into the container so that Renovate can dynamically invoke "sidecar" images when needed
```

Comment:
it installs required tools on the fly by default.
it optionally can use docker to install tools inside sidecar image instead of the current image.